client/me/security-2fa-key/add.jsx

```diff
@@ -23,19 +25,24 @@ class Security2faKeyAdd extends React.Component {
 		registerRequests: PropTypes.object.isRequired,
 	};
 
+	state = {
+		error: false,
+	}
+	
 	componentDidMount = () => {
 		this.registerKey();
 	};
 
 	registerKey = () => {
+		this.setState( { error: false } );
 		webauthn
 			.register()
 			.then( data => {
 				debug( 'registered key with data', data );
 				this.keyRegistered();
 			} )
 			.catch( err => {
-				this.props.onCancel( err );
```

Comment:
@david-binda don't we need to still call the onCancel func so that we can call `addKeyCancel` from index.jsx?

Reply:
Never mind I get it it's doing `onCancel` when the message is dismissed! Awesome!